src/tests/twoWayTrip.test.js

```diff
@@ -22,77 +58,86 @@ const testTwoWayTrip = () => {
           done();
         });
     });
-    it('should return 200 on successful created request ', (done) => {
+    it('should return 201 on successful created request ', (done) => {
       chai
         .request(app)
         .post('/api/v1/trips/returnTrip')
-        .send(validTrip)
+        .send({
+          origin: 'Kigali',
+          destination: 'Kampala',
+          departureDate: '2023-01-11',
+          returnDate: '2023-01-15',
+          reason: 'Having fun',
+          accommodation: 'Z campus'
+        })
         .end((err, res) => {
-          expect(res.status).to.equal(200);
-          expect(res.body).to.have.property('message').that.equals('request created on success!');
-          expect(res.body).to.have.property('status').that.equals('pending');
+          expect(res.status).to.equal(201);
+          expect(res.body).to.have.property('message').that.equals('request created with success!');
+          expect(res.body.data).to.have.property('status').that.equals('pending');
+          done();
         });
-      done();
     });
-    it('should return 422 on successful created request ', (done) => {
+    it('should return 422 on invalid request body', (done) => {
       chai
         .request(app)
         .post('/api/v1/trips/returnTrip')
-        .send(sameDirection)
+        .send(requestWithMissedComponent)
         .end((err, res) => {
           expect(res.status).to.equal(422);
           expect(res.body).to.have.property('error');
         });
       done();
     });
-    it('should return 422 on invalid trip', (done) => {
+    it('should return 422 when similar origin and destination', (done) => {
       chai
         .request(app)
         .post('/api/v1/trips/returnTrip')
-        .send(invalidTrip)
+        .send(requestWithSimilarOriginAndDestination)
         .end((err, res) => {
           expect(res.status).to.equal(422);
-          expect(res.body).to.have.property('error').which.is.not.empty;
+          expect(res.body).to.have.property('error').that.equals('Origin has to differ from destination.');
         });
       done();
     });
-    it('should return 422 if selected date collides with the previous requested trip', (done) => {
-      const tripRequest = {
-        origin: 'Kigali',
-        destination: 'Kampala',
-        departureDate: '2020-02-20',
-        returnDate: '2020-03-20',
-        accommodation: 'XYZ campus',
-        reason: 'Having fun',
-        passportNumber: '12345677'
-      };
+    it('should return 422 if departure date > return date', (done) => {
       chai
         .request(app)
         .post('/api/v1/trips/returnTrip')
-        .send(tripRequest)
+        .send(requestWithDepartureDateSetAfterReturnDate)
         .end((err, res) => {
+          console.log('eororo', res.body);
```

Comment:
@T2Wil  Please remember to clean your code before submitting, remember to remove ` console.log` that are not necessary.

Reply:
Understood!